common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisPilotingManager.java

```diff
@@ -432,8 +435,7 @@ public boolean beginFlight(boolean autoLand, Optional<GlobalConsoleBlockEntity>
                 }
 
 
-                for (Player player:
-                        this.operator.getLevel().players()) {
+                for (Player player : this.operator.getLevel().players()) {
                     PlayerUtil.sendMessage(player, Component.translatable("A dragon prevents you from progressing to The End."), true);
```

Comment:
Looks like a remnant of someone else's code, but this should be a translation

Reply:
Nope. That's from me. Fixed.